libs/sr-ui/src/Character/CharacterAutocomplete.tsx

```diff
@@ -16,6 +17,7 @@ export function CharacterAutocomplete({
   charKey,
   setCharKey,
 }: CharacterAutocompleteProps) {
+  const { t } = useTranslation(['character', 'charNames_gen'])
```

Comment:
does this conflict with genshin names?

Reply:
This should be fine, as the gh-pages filetree will be like
```
root
  genshin-optimizer
    assets
      charNames_gen.json
  starrail-optimizer
    assets
      charNames_gen.json
```